.github/workflows/ci-cd-staging.yml

```
@@ -105,7 +110,7 @@ jobs:
   publish-schema-npm:
     name: Publish schema npm package
     needs: [check-for-changes, deploy-apps]
-    if: ${{ always() && !failure() && !cancelled() && needs.check-for-changes.outputs.hasSwaggerSchemaChanges == 'true' || needs.check-for-changes.outputs.hasGqlSchemaChanges == 'true' }}
+    if: ${{ always() && !failure() && !cancelled() && needs.check-for-changes.outputs.hasSwagggerSchemaChanges == 'true' || needs.check-for-changes.outputs.hasGqlSchemaChanges == 'true'}}
```

actionlint (1.7.4)

property "hasswagggerschemachanges" is not defined in object type {hasbackendchanges: string; hasgqlschemachanges: string; hasinfrachanges: string; hasmigrationchanges: string; hasslacknotifierchanges: string; hasswaggerschemachanges: string; hastestchanges: string}

(expression)